README.md

```diff
@@ -811,6 +812,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [minio-java](https://github.com/minio/minio-java) - Provides simple APIs to access any Amazon S3 compatible object storage server.
 * [Protégé](http://protege.stanford.edu/) - Provides an ontology editor and a framework to build knowledge-based systems.
 * [Underscore-java](https://github.com/javadev/underscore-java) - Port of Underscore.js functions.
+* [strman-java](https://github.com/shekhargulati/strman-java): A Java 8 library for working with String.
```

Comment:
If we do accept strman-java, it must inserted in alphabetical order with the other entries around it. Please fix.